inferno-lsp/app/Main.hs

```diff
@@ -3,13 +3,14 @@
 
 module Main where
 
+import Control.Monad.Except (ExceptT)
 import Inferno.Eval.Error (EvalError)
 import Inferno.LSP.Server (runInfernoLspServer)
 import Inferno.Module.Prelude (builtinModules)
 import System.Exit (ExitCode (ExitFailure), exitSuccess, exitWith)
 
 main :: IO ()
 main = do
-  runInfernoLspServer @() @(Either EvalError) builtinModules >>= \case
+  runInfernoLspServer @() @(ExceptT EvalError IO) builtinModules >>= \case
```

Comment:
I find the `ExceptT err IO` stack a bit redundant, as `IO` already has exception capabilities. It might also be deceiving, as one might write a `case` for the `Either` returned by `runExceptT` and think they already dealt with any possible exception.